ble/GattClient.h

```diff
@@ -305,6 +306,53 @@ class GattClient {
     }
 
     /**
+     * @brief launch discovery of descriptors for a given characteristic
+     * @details This function will discover all descriptors available for a 
+     * specific characteristic. 
+     * 
+     * @param characteristic The characteristic targeted by this discovery
+     * @param callback This is the application callback for each descriptors 
```

Comment:
mismatched parameter name in the doc. Also please add doc for the third parameter.